third_party/terraform/resources/resource_bigtable_instance.go

```diff
@@ -366,3 +409,53 @@ func resourceBigtableInstanceClusterReorderTypeList(diff *schema.ResourceDiff, m
 
 	return nil
 }
+
+func resourceResize(diff *schema.ResourceDiff, meta interface{}) error {
+	old_count, new_count := diff.GetChange("cluster.#")
+	if old_count.(int) != new_count.(int) {
+		log.Printf("[DEBUG] resize from %d to %d", old_count, new_count)
+	}
+
+	new_clusters := make(map[string]interface{}, new_count.(int))
+	old_clusters := make(map[string]interface{}, old_count.(int))
+
+	for i := 0; i < new_count.(int) || i < old_count.(int); i++ {
```

Comment:
Do you mind commenting the blocks like this with what they're doing? While it's possible to figure out, customizeDiffs are pretty finicky and it's a big help to be able to read quickly what they're doing.